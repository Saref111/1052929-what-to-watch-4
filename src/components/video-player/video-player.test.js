import React from "react";
import renderer from "react-test-renderer";
import VideoPlayer from "./video-player.jsx";

const cover = `cover`;

test(`Videoplayer snaptest`, () => {
  const tree = renderer.create(<VideoPlayer
    isPlaying={false}
    poster={cover}
    preview={`1234`}
  />, {createNodeMock: () => {
    return {};
  }}).toJSON();

  expect(tree).toMatchSnapshot();
});
