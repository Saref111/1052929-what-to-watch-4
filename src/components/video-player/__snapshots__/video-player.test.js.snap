// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Videoplayer snaptest 1`] = `
<video
  className="player__video"
/>
`;
